### PR TITLE
Update locations.txt

### PR DESCRIPTION
**Pull request by ImreEzraToth**
Hi,
Id like to add my location to the file so i can use it in my projects.

kind regards,
Imre Toth

### CODE REVIEW

File: Tools/autotest/locations.txt [hunk 1]
```diff
@@ -57,3 +57,4 @@ Kawachi=35.879129,140.339683,7,0
 SpringValley= -35.280252,149.005821,597.3,5
 Pyramid=29.9764,31.1339,0,0
 AAMEastField=39.842288,-105.212928,1809,106
+Almere=52.365021, 5.201260, 0, 270
```
Comment (peterbarker) [inline Tools/autotest/locations.txt]: Does this actually work for you?  I didn't think the parser was actually lenient enough to allow spaces in there!

Note that you can *also* have a local set of flying sites:

```
pbarker@bluebottle:~$ cat /home/pbarker/.config/ardupilot/locations.txt
GSP=-35.207897,149.043640,582.840027,270
GSPM=-35.207745,149.043099,582.840027,270
MTRG=-35.2094851,149.0571052,595.800,90 # mount roger's school oval
MURN=-35.211195,149.040381,574.2,45 # mount roger's school oval
COBW=-35.620633,144.127053,247,135 # Oval on Cobwell Road
pbarker@bluebottle:~$ 
```

I'm not sure what the Windows equivalent of that is, sorry!

**Comment by ImreEzraToth** [comment]
Ooh I think something must’ve gone wrong there. I didn’t mean for the
spaces to be there :/ can I change them so it can still be in the folder?

Op za 15 feb. 2020 om 23:17 schreef Peter Barker <notifications@github.com>

**Comment by peterbarker** [comment]
@ImreEzraToth Oh, I'd merge it if it was valid.

I'm not *sure* you've thought about whether you *really* want it merged here when there is the option of having it locally.....

**Comment by ImreEzraToth** [comment]
Ooh sorry about that. I’m sort of new to the Linux world. Do you perhaps
know if there’s is an instruction soeverein on how to do it locally instead?
Thanks for your helpful responses.

Kind regards,
Imre Tóth


Op zo 16 feb. 2020 om 23:07 schreef Peter Barker <notifications@github.com>

**Comment by peterbarker** [comment]
> Ooh sorry about that. I’m sort of new to the Linux world. Do you perhaps know if there’s is an instruction soeverein on how to do it locally instead? Thanks for your helpful responses. Kind 

I've created a PR here: https://github.com/ArduPilot/ardupilot_wiki/pull/2539

Peter

**Comment by ImreEzraToth** [comment]
i was wondering if someone knows what i did wrong here because i get the error that the location doesnt exist, but i saved the location.
almere_locations

**Comment by peterbarker** [comment]
@ImreEzraToth is there a trailing carriage-return on that file?

Please show the invocation if it still fails.

**Comment by rmackay9** [comment]
If it's OK I think I'll close this PR.  Looks like it can be handled locally but happy to re-open if necessary

**Comment by ImreEzraToth** [comment]
thanks for the help :) I still haven't quite figured out how to fix it but
ill keep on trying


Op wo 19 feb. 2020 om 04:58 schreef Randy Mackay <notifications@github.com>:

**Comment by ImreEzraToth** [comment]
hi id created a totally new file under the right address and it still doesn't
Screenshot from 2020-02-22 11-41-44
 still wondering if there's a possibility to give me advise on the subject

**Comment by peterbarker** [comment]
> hi id created a totally new file under the right address and it still doesn't

Right filename, wrong location.  It should be in `~/.config/ardupilot/locations.txt`

> Screenshot from 2020-02-22 11-41-44
> still wondering if there's a possibility to give me advise on the subject

Sure.  Just post here.

**Pull request closed without merging**